README.md

```diff
@@ -87,8 +90,8 @@ const PeopleSearch = () = {
 ### Input
 
```

Comment:
Is it worth a note about which parameters are required and which optional?

Reply:
Done